### ISSUE TITLE
change strokeColor

### ISSUE THREAD

**Issue by mlight3** [efremidze/Magnetic#12]
Hi! 

I wanted to change strokeColor, but I can't.
I added below code , It works well.
```
    open func configure(text: String?, image: UIImage?, color: UIColor, strokeColor: UIColor) {
        self.text = text
        self.image = image
        self.color = color
        self.strokeColor = strokeColor
        maskOverlay = strokeColor
    }
```

But I want it makes more simpler, then I noticed that Node's `strokeColor` is same as maskOverlay's `strokeColor`.
I added property observation like this.
```
override open var strokeColor: UIColor {
        didSet {
              maskOverlay.strokeColor = strokeColor
        }
    }
```

But it doesn't work!
Cause self.frame is .zero when `maskOverlay` is evaluating. :(

I think there is a two way to handle this.
1. Use first approach
2. Use property observation, and remove lazy evaluation.

**Comment by efremidze** [comment]
Investigating

**Comment by efremidze** [comment]
Another solution:

```
node.strokeColor = .red
node.maskOverlay.strokeColor = .red
```

But Im looking for a better solution.

**Comment by efremidze** [comment]
Fixed in 1.0.6

**Comment by mlight3** [comment]
Looks good!
